src/ng/directive/attrs.js

```diff
@@ -159,7 +159,7 @@
  *
  * @description
  *
- * This directive sets the `disabled` attribute on the element if the
+ * This directive sets the `disabled` attribute on a form element if the
```

Comment:
[There are more][1] (e.g. optgroup, fieldset). Plus there is nothing stopping people from using it on elements that don't natively respond to `disabed`, such as custom controls. I wouldn't make too strong a statement (i.e. "it can be used on _these_ elements only). Mentioning the common usecase is fine though (either under the general term "form controls" or by explicit listing). 

[1]: https://www.w3.org/TR/html5/disabled-elements.html